lib/pluginmanager/util.rb

```diff
@@ -1,12 +1,35 @@
 require "rubygems/package"
 
 module LogStash::PluginManager
+
+  class Settings
```

Comment:
So to expand a bit on that, I think that if we want to introduce a global settings concept, this is not the right place to do that, we should think and discuss how we want to do that. for this PR purpose here I don't understand why a simple parameter for the sources is no ok?